src/main/java/io/netty/incubator/codec/http3/Http3ConnectionHandler.java

```diff
@@ -33,6 +33,9 @@
     private final LongFunction<ChannelHandler> unknownInboundStreamHandlerFactory;
     private final Http3ControlStreamInboundHandler localControlStreamHandler;
     private final Http3ControlStreamOutboundHandler remoteControlStreamHandler;
+    private final QpackHuffmanDecoder qpackHuffmanDecoder;
```

Comment:
Why do we need to store a reference here ? Shouldn'T this only be needed within the `QpackDecoder` ?

Reply:
I think I would store two instances just to make it more clear